### PR TITLE
Round up ideal width/height calculations so no control is cut off

### PR DESCRIPTION
**Pull request by carolhmj**
Related forum issue: https://forum.babylonjs.com/t/gui-adapt-tochildren-using-rounded-values-adaptive-scaling/29333

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13724/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13724/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13724/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13724/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13724/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13724/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13724/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13724/merge#BCU1XR#0

**Review by sebavan — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -350,6 +350,22 @@
             // "envFile": "${workspaceRoot}/.env",
             "preLaunchTask": "CDN Serve (Dev)"
         },
+        {
+            /*
```
Comment (sebavan) [inline .vscode/launch.json:354]: Should it be here or was it just debug ???

Comment (carolhmj) [reply .vscode/launch.json:354]: I added on purpose as I thought this could be useful to more people 😄

**Review by Popov72 — APPROVED** [review]

**Pull request merged**